packages/frontend/src/retro/components/dialogs/CreateRetroSessionDialog.tsx

```diff
@@ -100,26 +99,28 @@ export function CreateRetroSessionDialog() {
       <DialogTitle id="form-dialog-create-retro">Create Retro Session</DialogTitle>
       <DialogContent sx={{ display: "flex", flexDirection: "column", gap: "1rem" }}>
         <Box>
-          <DialogContentText>Please enter your name</DialogContentText>
-          <TextInput
-            value={name}
+          <UserNameInputField
+            userName={name}
+            id="user-name"
             onSubmit={handleSubmit}
             onChange={handleNameChange}
             error={isNameError}
-            id="user-name"
-            label="Username"
-            autoFocus
```

Comment:
We removed the autofocus here. Is that correct?

Reply:
No, it was a mistake that i moved it to the session name. Fixed it